.github/workflows/build.yml

```diff
@@ -101,15 +107,15 @@ jobs:
       - name: Build
         run: |
           python3 -m pip install --upgrade maturin
-          maturin build --release -o dist --target $RUST_MUSL_CROSS_TARGET --features unwind
+          maturin build --release -o dist --target ${{ matrix.target }} --features unwind
```

Comment:
Now that the tag matches the target, we can just use it directly which is a little clearer for someone not familiar with the container environment.